engines/sci/graphics/text16.cpp

```diff
@@ -340,6 +357,46 @@ int16 GfxText16::GetLongest(const char *&textPtr, int16 maxWidth, GuiResourceId
 	return resultCharCount;
 }
 
+// GetLongest() originally contained an off-by-one character bug which was
+//  fixed in late 1990. As part of this change, a width test was changed
+//  from greater than or equals to greater than. This method returns which
+//  behavior to use based on the current version and game.
+bool GfxText16::useEarlyGetLongestCalculations() {
```

Comment:
It would be better to move this inside features.cpp, like the rest of the version-specific feature checks